xbmc/addons/AddonSystemSettings.h

```diff
@@ -21,6 +21,12 @@ const int AUTO_UPDATES_ON = 0;
 const int AUTO_UPDATES_NOTIFY = 1;
 const int AUTO_UPDATES_NEVER = 2;
 
+enum class AddonTrustedUpdateMode : int
+{
+  PREFER_TRUSTED = 0,
```

Comment:
This should really be called TRUSTED_ONLY now I guess @howie-f ?

Reply:
definitely yes, put it on the list